src/Controls/ItemsControl.ts

```diff
@@ -202,8 +202,9 @@ module Fayde.Controls {
         }
         private _DisplayMemberTemplate: DataTemplate = null;
         private _GetDisplayMemberTemplate(): DataTemplate {

```

Comment:
This function was intended to be a memoize function.
Can we shift the logic to the consumers of this function?